x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/timeline_entry.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { EuiComment, EuiText } from '@elastic/eui';
+import { Immutable, EndpointAction } from '../../../../../../../common/endpoint/types';
+
+const TimelineItem = styled(EuiComment)``;
```

Comment:
Whats the purpose of this?

Reply:
Yeah, this is not entirely done. I want to add some bit of CSS to a pseudo-element to ensure the comment status header is similar to the one in the mock, where the time from now is separated with a `.`. The component doesn't allow for such a thing yet.

![Screenshot 2021-05-18 at 16 55 34](https://user-images.githubusercontent.com/1849116/118674174-e0baa480-b7f9-11eb-96cd-07e9cf462a39.png)

Reply:
I've removed the styled div for now. 2dd22ed92b3c54da9bdcce467dbe2d7e87097e4d. Will have a go at it in a later commit.